assets/app/index.html

```diff
@@ -65,7 +65,9 @@
               <li class="divider"></li>
               -->
               <li>
-                <a href="logout">Log out</a>
+                <!-- Stop event propagation to avoid a runtime error when the
+                     dropdown tries to close itself as we're navigating away. -->
+                <a href="logout" ng-click="$event.stopPropagation()">Log out</a>
```

Comment:
Yeah, it applies if `href` takes you somewhere. You would want the propagation to happen if `href=""` and you just have a click handler so that the menu closes.

I could stop the propagation in a handler, but we'd need to create one. I couldn't decide what was cleaner and simpler.